controllers/managedocs_controller.go

```diff
@@ -1752,6 +1759,41 @@ func (r *ManagedOCSReconciler) updateMCGCSV(csv *opv1a1.ClusterServiceVersion) e
 	return nil
 }
 
+func (r *ManagedOCSReconciler) reconcileConsolePlugin() error {
+	consolePlugin := &consolev1alpha1.ConsolePlugin{}
+	consolePlugin.Name = consolePluginName
+	consolePlugin.Namespace = r.namespace
+	err := r.get(consolePlugin)
+	if err != nil {
+		// ODF Operator has to bring this CR,
+		// so if not found, we silently return
+		if errors.IsNotFound(err) {
+			return nil
+		}
+		return err
+	}
+	expectedProxy := consolev1alpha1.ConsolePluginProxy{
+		Type:      consolev1alpha1.ProxyTypeService,
+		Authorize: true,
+		Alias:     consoleProxyAlias,
+		Service: consolev1alpha1.ConsolePluginProxyServiceConfig{
+			Name:      prometheusServiceName,
+			Namespace: r.namespace,
+			Port:      int32(templates.KubeRBACProxyPortNumber),
+		},
+	}
```

Comment:
We do not hard code configuration into a reconcile function, please create a template for this (inside the templates folder) and then deep copy it here

```suggestion
	expectedProxy := templates.ocpConsolePluginProxy.DeepClone()
```

Reply:
Done